recipes/pmw/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "Pmw" %}
+{% set version = "2.0.1" %}
+{% set sha256 = "0b9d28f52755a7a081b44591c3dd912054f896e56c9a627db4dd228306ad1120" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - Pmw
+
+about:
+  home: http://pmw.sourceforge.net/
+  license: BSD
```

Comment:
No problem. There is so much we can do on the packaging side.

This is good to go. I will merge as soon as the CIs are green.

Thanks!